README.md

```diff
@@ -16,12 +16,14 @@ Pack requires go version min. 1.11
 
 ## Configuration
 
+>_This plugin only works with clasic Slack apps at the moment. To learn how to create one, click [here](docs/classic-apps.md)._
```

Comment:
Small typo _clasic_ instead of _classic_

Reply:
Has been fixed